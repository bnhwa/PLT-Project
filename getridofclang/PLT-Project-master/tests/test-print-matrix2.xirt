num main() {
    xirtam m;
    m = [[1], [-3], [52.4155]];
    printm(m);
}