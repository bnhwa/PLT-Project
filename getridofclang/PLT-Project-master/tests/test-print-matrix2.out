1.00
-3.00
52.42